createdisk-library.sh

```diff
@@ -275,6 +275,9 @@ function generate_vfkit_bundle {
 
     generate_macos_bundle "vfkit" "$@"
 
+    create_qemu_image "$srcDir" "${SNC_PRODUCT_NAME}.qcow2" "microshift"
+    mv "${srcDir}/microshift" "${srcDir}/${SNC_PRODUCT_NAME}.qcow2"
```

Comment:
Is this correct for okd/openshift?

Reply:
@cfergeau (Good catch) no, It is only going to work with microshift, we might need another helper which execute this with `BUNDLE_TYPE` something `copy_qemu_image` like https://github.com/crc-org/snc/pull/749#discussion_r1245390062 or do you have a better helper name in mind?

Reply:
Maybe it's `create_qemu_image` which needs to be renamed to be more descriptive?

Reply:
or may be we should handle that bundle type as part of `create_qemu_image` function.

Reply:
`create_qemu_image` is relatively generic, adding some bundle type checks there would remove some of this genericity.

Reply:
> `create_qemu_image` is relatively generic, adding some bundle type checks there would remove some of this genericity.

@cfergeau agree, what we need a helper which encapsulate following code, because if we rename `create_qemu_image` and create 2 separate helper `create_qemu_image_snc` and `create_qemu_image_microshift` and call `create_qemu_image` internally for both helper then also this conditional block is required.
```bash
if [ $BUNDLE_TYPE != "microshift" ]; then
    create_qemu_image "$libvirtDestDir" "${VM_PREFIX}-base" "${VM_NAME}"
    mv "${libvirtDestDir}/${VM_NAME}" "${libvirtDestDir}/${SNC_PRODUCT_NAME}.qcow2"
else  
    create_qemu_image "$libvirtDestDir" "${VM_NAME}.qcow2" "microshift"
    mv "${libvirtDestDir}/microshift" "${libvirtDestDir}/${SNC_PRODUCT_NAME}.qcow2"
fi  
```